scss/bootstrap/overrides/style/_ui-select.scss

```diff
@@ -90,17 +96,18 @@
     }
 
     input {
-      border: 1px solid #727E8A !important;
+      border: 1px solid $input-border !important;
       box-shadow: none !important;
+      height: $input-height-base;
     }
 
     &:after {
       content: '\f002';
       bottom: 0;
       font-family: "FontAwesome";
       display: inline-block;
-      height: 33px;
-      line-height: 35px;
+      height: 100%;
```

Comment:
If I have a fixed height, everytime I need to change it if I change the height of the dropdown, but 100% works regardless